core/src/main/java/com/google/googlejavaformat/java/SnippetFormatter.java

```diff
@@ -103,7 +103,6 @@ public String createIndentationString(int indentationLevel) {
         throw new IllegalArgumentException(
             "comment formatting is only supported for compilation units");
       }
-      return formatter.getFormatReplacements(source, ranges);
```

Comment:
I think this would regress support for Eclipse's [`F_INCLUDE_COMMENTS`](https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.jdt.doc.isv%2Freference%2Fapi%2Forg%2Feclipse%2Fjdt%2Fcore%2Fformatter%2FCodeFormatter.html) option.